swarmform/scripts/sform_run.py

```diff
@@ -0,0 +1,200 @@
+import signal
+import traceback
+import json
+import yaml
+import six
+from argparse import ArgumentParser
+
+from fireworks.utilities.fw_serializers import recursive_dict, DATETIME_HANDLER
+from fireworks.scripts.lpad_run import init_yaml
+
+
+from swarmform import SwarmPad
+from swarmform.core.swarmwork import SwarmFlow
+from swarmform.core.cluster import cluster_sf
+from swarmform.sf_config import LAUNCHPAD_LOC, CONFIG_FILE_DIR
+
+DEFAULT_LPAD_YAML = "my_swarmpad.yaml"
+
+
+def handle_interrupt(signum):
+    sys.stderr.write("Interrupted by signal {:d}\n".format(signum))
+    sys.exit(1)
+
+
+def get_output_func(format):
+    if format == "json":
+        return lambda x: json.dumps(x, default=DATETIME_HANDLER, indent=4)
+    else:
+        return lambda x: yaml.safe_dump(recursive_dict(x, preserve_unicode=False),
+                                        default_flow_style=False)
+
+
+def get_sp(args):
+    try:
+        if not args.launchpad_file:
+            if os.path.exists(os.path.join(args.config_dir, DEFAULT_LPAD_YAML)):
+                args.launchpad_file = os.path.join(args.config_dir, DEFAULT_LPAD_YAML)
+            else:
+                args.launchpad_file = LAUNCHPAD_LOC
+
+        if args.launchpad_file:
+            return SwarmPad.from_file(args.launchpad_file)
+        else:
+            args.loglvl = 'CRITICAL' if args.silencer else args.loglvl
+            return SwarmPad(logdir=args.logdir, strm_lvl=args.loglvl)
+    except Exception:
+        traceback.print_exc()
+        err_message = 'SwarmForm was not able to connect to MongoDB. Is the server running? ' \
+                      'The database file specified was {}.'.format(args.launchpad_file)
+        if not args.launchpad_file:
+            err_message += ' Type "lpad init" if you would like to set up a file that specifies ' \
+                           'location and credentials of your Mongo database (otherwise use default ' \
+                           'localhost configuration).'
+        raise ValueError(err_message)
+
+
+def reset(args):
+    sp = get_sp(args)
+    if not args.password:
+        if input('Are you sure? This will RESET {} workflows and all data. (Y/N)'.format(
+                sp.workflows.count()))[0].upper() == 'Y':
+            args.password = datetime.datetime.now().strftime('%Y-%m-%d')
+        else:
+            raise ValueError('Operation aborted by user.')
+    sp.reset(args.password)
+
+
+def add_sf(args):
+    sp = get_sp(args)
+    if args.dir:
+        files = []
+        for f in args.sf_file:
+            files.extend([os.path.join(f, i) for i in os.listdir(f)])
+    else:
+        files = args.sf_file
+    for f in files:
+        fwf = SwarmFlow.from_file(f)
+        if args.check:
+            from fireworks.utilities.dagflow import DAGFlow
+            DAGFlow.from_fireworks(fwf)
+        sp.add_sf(fwf)
+
+
+def get_sf(args):
+    if args.id:
+        get_sf_by_id(args)
+    if args.name:
+        get_sf_by_name(args)
```

Comment:
Don't we need to return these values?

Reply:
These are only getting printed.No need to written anything. I will send an update with printing functionality